src/js/component/message.js

```diff
@@ -222,11 +228,15 @@ export default class MessageElement {
 
       /// message
       if (this.message.isAdminMessage()) {
+        // admin message
         this.messageFile.hide();
         this.media.hide();
         this.messageBox.show();
         this.messageText.html(this.message.message);
+        this.chatbotCard.hide();
```

Comment:
Is there a chance of this being empty?

Reply:
I think no, because it's rendered every time from the parser of above